app.rb

```diff
@@ -0,0 +1,130 @@
+require './person'
+require './student'
+require './teacher'
+require './book'
+require './classroom'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def start_console
+    puts 'welcome to w100 school Library App!'
+    until list_of_options
+      input = gets.chomp
+      if input == '7'
+        puts 'Thank You for using our school Library!'
+        break
+      end
+
+      option input
+    end
+  end
+
+  def list_all_books
+    puts 'Database is empty! Add a book.' if @books.empty?
+    @books.each { |book| puts "[Book] Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_persons
+    puts 'Database is empty! Add a person.' if @persons.empty?
+    @persons.each do |person|
+      puts "[#{person.class.name}] Name: #{person.name}, Age: #{person.age}, id: #{person.id}"
+    end
+  end
```

Comment:
After the above-mentioned issues are resolved kindly verify on the console to make sure that:

- You can successfully list all created persons
- You can create a rental with a permitted person
- You can list all rentals from a particular person using id

Well done on making the `create_book` and `list_all_books` methods work!!! 👏🏾